k8s/common-base/traefik/traefik.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: traefik
+  namespace: admin
+  annotations:
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: traefik
+  chart:
+    repository: https://kubernetes-charts.storage.googleapis.com/
+    name: traefik
+    version: 1.61.1
+  values:
+    imageTag: 1.7.9
+    resources:
+      requests:
+        cpu: 100m
+        memory: 20Mi
+      limits:
+        cpu: 1000m
+        memory: 300Mi
+    service:
+      annotations:
+        service.beta.kubernetes.io/azure-load-balancer-internal: "true"
```

Comment:
hmm I wonder if we can override this to empty in the customisation?
something like:
```
service:
  annotations: []
```
as demo uses a public load balancer

Reply:
Yes, tested in local , it works. Flux should just apply the result of Kustomise.